credentials/src/test/kotlin/web5/sdk/credentials/PresentationDefinitionTest.kt

```diff
@@ -0,0 +1,91 @@
+package web5.sdk.credentials
+
+import assertk.assertThat
+import assertk.assertions.contains
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+import com.networknt.schema.JsonSchemaFactory
+import com.networknt.schema.SpecVersion
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+
+class PresentationDefinitionTest {
+  val jsonMapper: ObjectMapper = ObjectMapper()
+    .registerKotlinModule()
+    .findAndRegisterModules()
+    .setSerializationInclusion(JsonInclude.Include.NON_NULL)
+
+  @Test
+  fun `can serialize`() {
```

Comment:
One additional test that I find very useful is around idempotency. See https://github.com/TBD54566975/web5-kt/blob/a8d10d9fa92226d2432dbb8fd85f9a46792bf5be/dids/src/test/kotlin/web5/sdk/dids/DidResolutionResultTest.kt#L13 for an example. It makes sure that data isn't lost in the round trip.